mlflow/tracing/export/mlflow.py

```diff
@@ -109,50 +99,3 @@ def _serialize_inputs_outputs(self, input_or_output: Optional[Dict[str, Any]]) -
             trunc_length = MAX_CHARS_IN_TRACE_INFO_ATTRIBUTE - len(TRUNCATION_SUFFIX)
             serialized = serialized[:trunc_length] + TRUNCATION_SUFFIX
         return serialized
-
-
-class InMemoryTraceDataAggregator:
```

Comment:
note: This class is moved to `InMemoryTraceManager` and standalone file, given the extension of responsibility. Now this in-memory registry handles trace info not only spans and serves as a key state management for imperative APIs.